plugins/Channel/config.py

```diff
@@ -63,4 +63,9 @@ def configure(advanced):
         be used (they are optional in the IRC protocol). The standard
         substitutions ($version, $nick, etc.) are all handled appropriately.""")))
 
+conf.registerGroup(Channel, 'invite')
+conf.registerChannelValue(Channel.invite, 'requireCapability',
+    registry.String('op', _("""Determines what capability (if any) the bot should
+    require people trying to use the 'invite' command to have.""")))
```

Comment:
Should we say explicitly that setting an empty value will allow anyone to use INVITE?